members-list.md

```diff
@@ -1103,5 +1103,9 @@ Not really that interesting of a person. I do enjoy the occasional video game (W
 **Link to Profile**: [Mrudhulraj](https://github.com/Mrudhulraj)  <br/>
 **About Me**:  Just a person who admires NLP and Image processing. Tbh want to contribute a lot to Open Source and active user in Medium(https://medium.com/@mrudhulraj.n) Btw huge bball fan and interested a little in photography.<br/>
 
+**Name**: Amritesh <br/>
+**SRN**: PES1UG19CS056 <br/>
+**Link to Profile**: https://github.com/ammrit2312  <br/>
```

Comment:
Hi, thanks for your PR!
Please create a link to your account through your username.
For example: [MadhavJivrajani](https://github.com/MadhavJivrajani) and not https://github.com/MadhavJivrajani

You can do this by doing \[github user name](link to github account)